### DISCUSSION TITLE
Build from sources

### DISCUSSION THREAD

**Discussion by sourcecodeit** [vuejs/pinia#2735] [Help and Questions] [answered]
Hi,

I'd like to contribute to this project, however, I don't have experience with Github workspaces, can you tell me the steps?

I currently cloned the repository and tried to install packages with `npm i` but I get this error:

```
npm ERR! code EDUPLICATEWORKSPACE
npm ERR! must not have multiple workspaces with the same name
npm ERR! package '@pinia/playground' has conflicts in the following paths:
npm ERR!     /Users/darko/Code/pinia/packages/online-playground
npm ERR!     /Users/darko/Code/pinia/packages/playground
```
I'm using:

```
➜  pinia git:(v2) npm -v
9.8.1
➜  pinia git:(v2) node -v
v18.18.2

```

**Comment by posva** [comment, marked as answer]
You don't need github workspaces. Clone the repository locally and use pnpm to install dependencies. If you have corepack enabled, it should even download the right versions for you

**Reply by sourcecodeit** [reply]
Thank you.. I was stupidly using npm -.-
